protocol/streaming/full_node_streaming_manager.go

```diff
@@ -424,9 +523,47 @@ func (sm *FullNodeStreamingManagerImpl) SendTakerOrderStatus(
 	)
 }
 
-// AddUpdatesToCache adds a series of updates to the full node streaming cache.
+// SendSubaccountUpdates groups subaccount updates by their subaccount ids and
+// sends messages to the subscribers.
+func (sm *FullNodeStreamingManagerImpl) SendSubaccountUpdates(
+	subaccountUpdates []satypes.StreamSubaccountUpdate,
+	blockHeight uint32,
+	execMode sdk.ExecMode,
+) {
+	defer metrics.ModuleMeasureSince(
+		metrics.FullNodeGrpc,
+		metrics.GrpcSendSubaccountUpdatesLatency,
+		time.Now(),
```

Comment:
Can you update the datadog dashboards with subaccount metrics? would be good to know what % of the messages in our update cache are subaccounts, etc

Reply:
created https://linear.app/dydx/issue/CT-1116/update-datadog-dashboards-for-subaccount-fns to track